### DISCUSSION TITLE
grouping pdf files

### DISCUSSION THREAD

**Discussion by nilnil47** [ahrm/sioyek#1001] [Q&A] [answered]
hey,
first of all, thank you very much for all your work!
Sioyek is a great product!

I am using Sioyek as my PDF viewer for my university courses and I wanted to ask if there is a feature that behaves similarly to the following description:
for each class, I have multiple pdf that are associated with that class and they are different from each other. I would be really happy if there were the option to choose a subject and then only the associated to that PDF of the subject would be displayed in the recent view.

Thank you very much!

**Comment by ahrm** [comment, marked as answer]
There is no feature exactly like that. You could bookmark all the documents for a class with a distict bookmark like : `[class name] document name` and then search for `[class name]` in the global bookmark list.

**Reply by nilnil47** [reply]
cool, thank you very much!
